15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md

```diff
@@ -272,6 +272,10 @@ The deprecated version, `Microsoft.Data.SQlite`, has been removed and will requi
 
 ```
 
+* **Webhook payload property casing has changed**
+
+Following changes to serialization, please be aware that the properties in the payload now use camel case, rather than pascal case. For example the 'CreateDate' property becomes 'createDate'.
```

Comment:
```suggestion
 Following changes to serialization, property names in the payload now use camelCase instead of PascalCase. For example, the  'CreateDate' property is now 'createDate'.
```